exampleSite/content/docs/guide/markdown.fa.md

```
@@ -1,141 +1,195 @@
 
-|     نام | سن  |
-|--------|------|
-|    گودرز |  ۳۰|
-|  آصف |    ۳۴  |
+هشدارها یک افزونه Markdown بر اساس نحو نقل‌قول هستند که می‌توانید برای تأکید بر اطلاعات مهم از آن‌ها استفاده کنید.
+[هشدارهای به سبک GitHub](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) پشتیبانی می‌شوند.
+لطفاً مطمئن شوید که از آخرین نسخه Hextra و [Hugo v0.146.0](https://github.com/gohugoio/hugo/releases/tag/v0.146.0) یا بالاتر استفاده می‌کنید.
+
+> [!NOTE]
+> اطلاعات مفیدی که کاربران باید بدانند، حتی هنگام مرور سریع محتوا.
+
```

markdownlint-cli2 (0.17.2)

Blank line inside blockquote

(MD028, no-blanks-blockquote)